docs/reporting-standards.md

```diff
@@ -71,11 +83,220 @@ We propose the title should use the following syntax
+independent contributors. These agencies and the individual software
+developers of InaSAFE take no responsibility for the correctness of outputs
+from InaSAFE or decisions derived as a consequence.
+
+# Reports
+
+## Analysis question
+
+### Description
+
+The analysis question appears at the beginning of each report and identifies
+ and describes the purpose of the analysis. Each report should have an analysis
+ question. It should add more detail to the existing map title. The question
+ should reflect the terminology used in the title. Using the below table as a
+ reference, the questions should use the following syntax:
+
+ In the event of a <**hazard type**>, <**exposure measure**><**exposure type**>
+ might be affected?
+
+ or for generic impact functions:
+
+ In each of the hazard zones <**exposure measure**> <**exposure type**> might be
+ affected?
+
+![image](https://cloud.githubusercontent.com/assets/16660099/17200614/1f492106-54b1-11e6-8ac8-19d0b9ac9e3b.png)
+
+Questions should always have a question mark at the end of the sentence.
+
+### Examples
+
+- In the event of a flood, how many people might be affected?
+- In the event of an earthquake, which land cover might be affected?
+- In each of the hazard zones how many buildings might be affected?
+- In each of the hazard zones which length of road might be affected?
+
+## General report
+
+The general report gives a brief summary in table form of what the estimated
+effect of the hazard will be. It should follow this form:
+Estimated <**exposure measure**><**exposure type**>
+
+![image](https://cloud.githubusercontent.com/assets/16660099/17165600/a45ffea2-53fb-11e6-91c3-761157dd7d8d.png)
+
+## Detailed report
+
+The detailed report gives further detail on the estimated effect of the hazard,
+broken down into characteristics, e.g. gender and age for population, type of
+building for infrastructure.
+
+![image](https://cloud.githubusercontent.com/assets/16660099/17165692/739dd55e-53fc-11e6-82db-eede1413d6e4.png)
+
+## Minimum needs report
+
+The minimum needs report gives information on the estimated amount of relief
+ items to support the affected population. It should be broken up into relief
+ items that will be provided once and relief items that are to provided weekly.
+
+![image](https://cloud.githubusercontent.com/assets/16660099/17166008/86065ffc-53fe-11e6-841e-cea3b81803f6.png)
+
+## Action checklist
+
+### Description
+
+Action checklists suitable for each hazard type and exposure element should be
+ included in this section. Each report should have an action checklist. These
+ checklists include questions that should be asked in the event of a hazard
+ occurring and should reflect the exposure type analysed and the results of
+ the report. The checklist should be sorted in the following order: general
+ information, hazard specific questions, exposure specific questions.
+ questions related to minimum needs (if applicable) and questions regarding
```

Comment:
does the checklist have all these components?
Or is it more exposure specific questions?